Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -78,7 +88,14 @@ private int Start(string file, bool debug)
             }
             else if (debug && chromeProtocolRequired)
             {
-                StartAndAttachDebugger(file, nodePath);
+                if (CheckUseNewChromeDebugProtocolOption())
+                {
+                    StartWithChromeV2Debugger(file);
```

Comment:
You don't pass in the node path, so which version are you using? 

We allow the user to specify a specific version of node in the project properties, so we should use that.